method/method.go

```diff
@@ -124,6 +126,12 @@ func New() *Method {
 		wg:         &wg,
 		stdout:     log.New(os.Stdout, "", 0),
 	}
+	sess, _ := session.NewSession(aws.NewConfig())
```

Comment:
It seems that these lines make the assumptions that:
1. The apt process is running on an instance in EC2.
1. The instance is in the same region as the s3 bucket where the apt repository is hosted.

Do I understand that correctly?

Reply:
Yes, that will save effort to set region in apt config when bucket is same region with instance.